plugins/logging/pkg/gateway/admin_v2.go

```diff
@@ -1008,6 +1012,7 @@ FETCH:
 }
 
 func (m *LoggingManagerV2) generateAdminPassword(cluster *loggingv1beta1.OpniOpensearch) (password []byte, retErr error) {
+	ctx := context.TODO()
```

Comment:
Since this is the plugin-side external reconciler, does it makes sense to pass in the Logging plugin ctx to the `LoggingManagerV2` and use this as a context here?

Reply:
possibly but I think the use of contexts, and the passing in of them needs careful review.  This is intended as a stopgap pending that review which is why it's `context.TODO()` instead of `context.Background()`